server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -431,6 +431,7 @@ private static ObjectId fetchRemoteHeadAndGetCommitId(
             GitWithAuth git, String headBranchRefName) throws GitAPIException, IOException {
         final FetchCommand fetch = git.fetch(1);
         final FetchResult fetchResult = fetch.setRefSpecs(new RefSpec(headBranchRefName))
+                                             .setCheckFetchedObjects(true)
                                              .setRemoveDeletedRefs(true)
```

Comment:
Hi @jrhee17, because we only fetch 1, so if the `setCheckFetchedObjects(true)`, it will throw exeption `complete object graph`

Reply:
Just ran into this myself. Now I know why 😄    Thanks for the explanation 👍